packages/vitest/src/integrations/chai/jest-matcher-utils.ts

```diff
@@ -166,6 +164,8 @@ export interface DiffOptions {
 
 // TODO: do something with options
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-export function diff(a: any, b: any, options?: DiffOptions) {
+export async function diff(a: any, b: any, options?: DiffOptions) {
```

Comment:
this function cannot be async, it is used by `extended` jest matchers. `@testing-library` for example will not work as expected